File: src/com/runelive/model/input/impl/ThievBots.java
package com.runelive.model.input.impl;

import com.runelive.model.input.Input;
import com.runelive.world.content.skill.impl.thieving.Stalls;
import com.runelive.world.entity.impl.player.Player;

public class ThievBots extends Input {

	@Override
	public void handleSyntax(Player player, String syntax) {
		player.getPacketSender().sendInterfaceRemoval();
		if (Stalls.getBotStop() == 1) {
			if (syntax.equalsIgnoreCase("ikov2")) {
				player.getPacketSender().sendMessage("You have successfully completed the random.");
				player.setPassedRandom(true);
			} else {
				player.getPacketSender().sendMessage("That was not the right answer, please try again.");
				player.setPassedRandom(false);
				// player.getPacketSender().sendLogout();
			}
		} else if (Stalls.getBotStop() == 2) {
			if (syntax.equalsIgnoreCase("15")) {
				player.getPacketSender().sendMessage("You have successfully completed the random.");
				player.setPassedRandom(true);
			} else {
				player.getPacketSender().sendMessage("That was not the right answer, please try again.");
				player.setPassedRandom(false);
				// player.getPacketSender().sendLogout();
			}
		} else if (Stalls.getBotStop() == 3) {
			if (syntax.equalsIgnoreCase("no")) {
				player.getPacketSender().sendMessage("You have successfully completed the random.");
				player.setPassedRandom(true);
			} else {
				player.getPacketSender().sendMessage("That was not the right answer, please try again.");
				player.setPassedRandom(false);
				// player.getPacketSender().sendLogout();
			}
		}
	}
}

File: src/org/scripts/kotlin/content/skills/thieving/ThievBots.kt
package com.scripts.kotlin.content.skills.thieving;

import com.runelive.model.input.Input
import com.runelive.world.entity.impl.player.Player
import org.scripts.kotlin.content.skills.thieving.Stall

class ThievBots : Input() {

    override fun handleSyntax(player: Player, syntax: String) {
        player.packetSender.sendInterfaceRemoval()
        if (Stall.botStop == 1) {
            if (syntax.equals("ikov2", ignoreCase = true)) {
                player.packetSender.sendMessage("You have successfully completed the random.")
                player.isPassedRandom = true
            } else {
                player.packetSender.sendMessage("That was not the right answer, please try again.")
                player.isPassedRandom = false
                // player.getPacketSender().sendLogout();
            }
        } else if (Stall.botStop == 2) {
            if (syntax.equals("15", ignoreCase = true)) {
                player.packetSender.sendMessage("You have successfully completed the random.")
                player.isPassedRandom = true
            } else {
                player.packetSender.sendMessage("That was not the right answer, please try again.")
                player.isPassedRandom = false
                // player.getPacketSender().sendLogout();
            }
        } else if (Stall.botStop == 3) {
            if (syntax.equals("no", ignoreCase = true)) {
                player.packetSender.sendMessage("You have successfully completed the random.")
                player.isPassedRandom = true
            } else {
                player.packetSender.sendMessage("That was not the right answer, please try again.")
                player.isPassedRandom = false
                // player.getPacketSender().sendLogout();
            }
        }
    }
}
